penguin/web/routes.py

```
@@ -2178,6 +2150,32 @@ async def list_skills(
     return payload
 
 
+@router.get("/api/v1/mcp")
+async def get_mcp_status(
+    refresh: bool = False,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)